react/features/toolbox/components/web/Toolbox.js

```diff
@@ -1093,13 +1108,20 @@ class Toolbox extends Component<Props, State> {
                 );
             case 'closedcaptions':
                 return <ClosedCaptionButton showLabel = { true } />;
-            case 'info':
-                return <InfoDialogButton showLabel = { true } />;
+            case 'security':
+                return (
+                    <OverflowMenuItem
+                        accessibilityLabel = { t('toolbar.accessibilityLabel.security') }
+                        icon = { this.props._locked ? IconLockPassword : IconUnlockPassword }
+                        key = 'security'
+                        onClick = { this._onToolbarOpenSecurity }
+                        text = { t('toolbar.security') } />
+                );
             case 'invite':
                 return (
                     <OverflowMenuItem
                         accessibilityLabel = { t('toolbar.accessibilityLabel.invite') }
-                        icon = { IconInvite }
```

Comment:
I may be wrong, but I don't see this icon used anywhere else. If it's true, can you please remove the actual icon too?